grails-app/services/au/org/ala/biocache/hubs/PostProcessingService.groovy

```diff
@@ -505,6 +505,10 @@ class PostProcessingService {
     }
 
     def processUserFQInteraction(SpatialSearchRequestParams requestParams, activeFacetObj) {
+        if (requestParams.disableAllQualityFilters) {
+            return [[:], [:], [:], [:]]
```

Comment:
Seems a big ugly to return 4 maps... would prefer a bean to be used as data transport class if data is this complex.

Reply:
Hi Nick, how about just a map.
so it's like

```
def result = [:]
if (disableAll) {
    return result
} 

result.val1 = val1
result.val2 = val2

return result
```

Reply:
Yeah, a plain Map is OK